### ISSUE TITLE
Poor canvas performance (drawImage) caused by electron

### ISSUE THREAD

**Issue by TickleThePanda** [zeit/hyper#2790]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 (10.0.16299 Build 16299)
- **Hyper.app version**: 2.0.0-canary15
- **The issue is reproducible in vanilla Hyper.app**
- **Relevant information from devtools**: 
![image](https://user-images.githubusercontent.com/13658288/38018547-158047e4-326d-11e8-8e4d-f9cb14e41018.png)

## Issue
Poor performance due to electron bug: electron/electron#11051

**Comment by bgw** [comment]
Yep. The electron issues probably don't help, but it's slow on up-to-date versions of Chrome too.

Xterm.js often draws much more to the screen than it needs to, mostly because of some complications with overlapping wide characters. I'm working towards fixing this, and some other related performance issues, so the idea is that while it'll still be possible to end up with 160 ms frames, they'll be much less common.

I think if we want to completely eliminate these kinds of performance issues, we'll need to use WebGL, since that'll let us fill the entire screen with one or two draw calls. However, that's a lot more complicated, so I'm not interested in working on it the moment.

**Comment by Stanzilla** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=683994
